adcc/ExcitedStates.py

```diff
@@ -657,6 +659,66 @@ def describe_amplitudes(self, tolerance=0.01, index_format=None):
                 ret += separator
         return ret[:-1]
 
+    def to_dataframe(self):
+        """
+        Exports the ExcitedStates object as :class:`pandas.DataFrame`.
+        All values are printed in atomic units.
```

Comment:
printed? I'd just say atomic units are used for all values.

Reply:
Done.